DebugGUI/src/DebugGUI.h

```diff
@@ -19,6 +20,10 @@ void default_error_callback(int, const char*);
 
 void* initGUI(const char* name, decltype(default_error_callback) = nullptr);
 bool pollGUI(void* context, std::function<void(void)> guiCallback);
+void getFrameJSON(void *data, std::ostream& json_data);
```

Comment:
This you should be able to make static.

Reply:
My idea was to call it in O2 to get the json data from the draw data and then send it to the remote GUIs.

Reply:
nevermind, I missed I do not have a wrapper class for that.